schema/ddl/20240312171939_create_itembarcode.sql

```diff
@@ -0,0 +1,19 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS itemBarcodes (
```

Comment:
[must] スネークケースでお願いします！
```suggestion
CREATE TABLE IF NOT EXISTS item_barcodes (
```